src/bundle/Resources/views/content/tabs/versions.html.twig

```diff
@@ -16,9 +24,18 @@
 {% endif %}
 
 {% if archivedVersions %}
+    {{ form_start(archivedActionsForm, {'attr': {'class': 'ez-js-standard-form'}}) }}
+
     <div class="ez-list-toolbar">
         <h2 class="ez-list-toolbar-label">{{ 'locationview.versions.archived.versions'|trans()|desc('Archived versions') }}</h2>
+        <div class="container-buttons">
+            {{ form_row(archivedActionsForm.new_draft, {'label': 'locationview.versions.new.draft.selected.version'|trans()|desc('New draft based on selected version'), 'attr': {'class': 'ez-button ez-button-secondary'}}) }}
```

Comment:
Symfony forms are quite extensible so it can be removed if needed: https://symfony.com/doc/current/form/form_customization.html
I presume it will need removing from all buttons throughout the repo?